rules/apple_patched.bzl

```diff
@@ -84,7 +84,7 @@ def _apple_framework_import_modulemap_impl(ctx):
     # is to list here the keys to all of them (you can see the keys for the existing providers of a
     # target by just printing the target)
     # For more information refer to https://groups.google.com/forum/#!topic/bazel-discuss/4KkflTjmUyk
-    other_provider_keys = [AppleFrameworkImportInfo, SwiftUsageInfo, apple_common.AppleDynamicFramework, OutputGroupInfo, DefaultInfo]
+    other_provider_keys = [AppleFrameworkImportInfo, SwiftUsageInfo, apple_common.AppleDynamicFramework, OutputGroupInfo, DefaultInfo, AppleResourceInfo]
```

Comment:
This is just a quick fix: the AppleResourceInfo was not being propagated correctly